shared/chat/conversation/messages/wrapper/exploding-height-retainer/index.js.flow

```diff
@@ -5,6 +5,7 @@ import type {StylesCrossPlatform} from '../../../../../styles'
 export type Props = {
   children?: React.Node,
   explodedBy: ?string,
+  exploding: boolean,
```

Comment:
@chrisnojima `messageHeights` had 1500+ keys after scrolling up for a minute in chat if we don't limit it to exploding messages.